### PR TITLE
Add a tool to record and replay sessions

### PR DESCRIPTION
**Pull request by mathieu-lemay**

### CODE REVIEW

File: f1-telemetry-display/src/main.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ async fn main() {
             log_builder.with_term_logger(LevelFilter::Info, TerminalMode::Mixed, ColorChoice::Auto);
     }
 
-    log_builder.build().expect("Error initializing loggger.");
+    log_builder.build().expect("Error initializing logger.");
```
Comment (hortonhearsadan) [inline f1-telemetry-display/src/main.rs:72]: eurgh driveby refactor of a log line, out of scope, DECLINED

**Review by hortonhearsadan — APPROVED** [review]
rawe ceek

File: f1-telemetry-display/src/main.rs (hunk 1 above)
Comment (mathieu-lemay) [reply f1-telemetry-display/src/main.rs:72]: bruh

**Review by hortonhearsadan — APPROVED** [review]
